modules/general/container-app-environment-with-config-publish/main.json

```diff
@@ -258,9 +258,13 @@
               "properties": {
                 "Application_Type": "web",
                 "Flow_Type": "Redfield",
-                "Request_Source": "CustomDeployment"
```

Comment:
Shouldn't there be a corresponding main.bicep change for this? I only see main.json in the "changed files" tree.

Reply:
Not in this case - the `.json` files are generated by the local build process.  The reason this ARM template has changed is because it has a dependency on the one that was updated.

(i.e. When generating the `.json` ARM templates any module references get resolved at that time and their contents baked into the ARM output)